modules/storageAvailable.js

```diff
@@ -0,0 +1,18 @@
+export default function storageAvailable(type) {
+  // Check if Storage is available
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      e.code === 22
+        || e.code === 1014
+        || e.name === 'QuotaExceededError'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      && (storage && storage.length !== 0);
+  }
+}
```

Comment:
- [ ] Great job for the storageAvailable(type)  function implementation, :clap: but I think it will be better if you can use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)